python/lsst/daf/butler/core/datasets.py

```diff
@@ -57,6 +56,19 @@ def name(self):
         """
         return self._name
 
+    @property
+    def dataUnits(self):
+        """A `frozenset` of `DataUnit` names that defines the `DatasetRef`\ s
+        corresponding to this `DatasetType`.
+        """
+        return self._dataUnits
+
+    @property
+    def storageClass(self):
+        """A `StorageClass` that defines how this `DatasetType` is persisted.
+        """
+        return self._storageClass
+
     @property
     def template(self):
         """A string with `str`.format-style replacement patterns that can be
```

Comment:
This is not related to the file name template used by datastore. Correct? (so this template is not a `FileTemplate`).

Reply:
It is related to that template. But it is unclear (to me at the moment) if we still want it here. I'll remove it for now.